dockerutil/errors.go

```diff
@@ -0,0 +1,13 @@
+package dockerutil
+
+import "fmt"
+
+func HandleNodeJobError(exitCode int, _, _ string, err error) error {
+	if err != nil {
+		return err
+	}
+	if exitCode != 0 {
+		return fmt.Errorf("container returned non-zero error code: %d\n", exitCode)
```

Comment:
The two unnamed string arguments are standard output and error, so those should be part of the formatted error message. Or maybe use a new structured error for this, in case we need to introspect the error somewhere else.

Reply:
This function was copied. I split it into a separate file `errors.go` to separate from the strings helpers. I agree a function with unused args is suspect. In this case, it's a convenience given the return signature in other areas of the codebase. 

It probably warrants a refactor but I would argue out of scope for this PR.

Reply:
The NodeJob method returns stdout and stderr so that command callers can use them if necessary to parse output. Execute-only commands that don't need to parse the output can simply wrap the NodeJob call with this method to get an error return. But I don't think that stderr (or stdout) should be printed here necessarily. I had it printing stderr when err != nil and it made the test very hard to follow since some containers have extremely verbose stderr. But maybe printing stderr for that case is fine now that the long running chain node container output is disabled unless `SHOW_CONTAINER_LOGS` env var is provided. 

> It probably warrants a refactor but I would argue out of scope for this PR.

+1